docs/specification/current.md

```diff
@@ -268,9 +268,8 @@ No single FHIR resource represents a user, rather Practitioner and PractitionerR
 
 To reduce the implementation burden on CDS Clients that support CDS Services, this specification RECOMMENDS that prefetch queries only use a subset of the full functionality available in the FHIR specification. Valid prefetch templates SHOULD only make use of:
 
-* _instance_ level [read](https://www.hl7.org/fhir/http.html#read) interactions (for resources with known ids such as `Patient` and `Practitioner`)
+* _instance_ level [read](https://www.hl7.org/fhir/http.html#read) interactions (for resources with known ids such as `Patient`, `Practitioner`, or `Encounter`; e.g. `patient={{context.patientId}}`)
 * _type_ level [search](https://www.hl7.org/fhir/http.html#search) interactions
-* Patient references (e.g. `patient={{context.patientId}}`)
```

Comment:
Do we know the history on this line?  I'm trying to understand what kind of constraints it's meant to communicate. No other resource references in a type search?